ssn/chapters/Observation.html

```diff
@@ -440,6 +441,92 @@ <h5><a href="#SOSAObservationCollection">sosa:ObservationCollection</a></h5>
         </div>
     </section>
 
+    <section class="informative specterm sosa" id="SOSAObservingProcedure" about="sosa:ObservingProcedure"
+        typeof="owl:Class">
+        <h6><a href="#SOSAObservingProcedure">sosa:ObservingProcedure</a></h6>
+        <p class="crossreference">
+            <strong>IRI:</strong> http://www.w3.org/ns/sosa/ObservingProcedure
+        </p>
+        <p>
+            <strong>an OWL Class</strong>
+        </p>
+        <p>
+            <em property="rdfs:label">Observing Procedure</em> -
+            <span property="skos:definition"> A workflow, protocol, plan, algorithm, or computational method specifying
+                how to
+                make an Observation.
+                An Observing Procedure is re-usable, and might be involved in many Observations.
+                It explains the steps to be carried out to arrive at reproducible results.
+            </span>
+        </p>
+        <table>
+            <tbody>
+                <tr class="ssn">
+                    <th>Sub class of</th>
+                    <td>
+                        <span rel="rdfs:subClassOf" resource="sosa:Procedure">sosa:Procedure</span>
+                    </td>
+                </tr>
+                <tr class="ssn">
+                    <th class="restrictions">Restriction</th>
+                    <td>
+                        <span about="sosa:ObservingProcedure" rel="rdfs:subClassOf"
+                            resource="[_:SOSAObservingProcedure1]"></span>
+                        <span about="[_:SOSAObservingProcedure2]" typeof="owl:Restriction"></span>
+                        <span about="[_:SOSAObservingProcedure2]" rel="owl:onProperty" resource="sosa:implementedBy"><a
+                                href="#SOSAimplementedBy">sosa:implementedBy</a></span>
+                        <span about="[_:SOSAObservingProcedure2]" property="owl:allValuesFrom"
+                            resource="sosa:Sensor"><em><b>ONLY</b></em> <a href="#SOSASensor">sosa:Sensor</a></span>
+
+                    </td>
+                </tr>
+                <tr>
+                    <th>Examples</th>
+                    <td>
+                        <span property="skos:example">A workflow, protocol, plan, algorithm, or computational method
+                            specifying how to make an observation; the description of the process utilized by an
+                            observer.
+                            This could be a chemical analysis method, a protocol for measuring an object, but could also
+                            be a checklist utilized by a human observer during a biodiversity campaign.
```

Comment:
```suggestion
                            This could be a chemical analysis method, a protocol for measuring an object, or even
                            a checklist used by a human observer during a biodiversity campaign.
```